app/src/main/java/com/epfl/beatlink/MainActivity.kt

```diff
@@ -23,6 +23,7 @@ import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.navigation
 import androidx.navigation.compose.rememberNavController
+import com.android.sample.ui.library.LibraryScreen
```

Comment:
The import doesn't match the package structure.

```suggestion
import com.epfl.beatlink.ui.library.LibraryScreen
```